osc-server/src/main/java/org/osc/core/broker/service/securitygroup/BindSecurityGroupService.java

```diff
@@ -405,9 +406,12 @@ private ServiceFunctionChain validateAndLoadSfcBind(EntityManager em, BindSecuri
              VirtualSystem vs = validateAndLoadVirtualSystem(em, virtualSystemId);
              
              //if this virtual system is pointing to more than one SFC and one of the SFC other than given SFC
-             //is already binded to SG and active, throw exception
-            if (vs.getServiceFunctionChains().size() > 1) {
-                for (ServiceFunctionChain serviceFunctionChain : vs.getServiceFunctionChains()) {
+             //is already binded to SG and active, throw exception;
+             //Get All the sfcs having this virutalsystem in the chain
```

Comment:
virtualsystem -> virtual system

Reply:
Will make this change if there is change request for this file..later.

Reply:
corrected the typo